python/tvm/relay/frontend/tensorflow.py

```diff
@@ -1599,6 +1599,9 @@ def _impl(inputs, attr, params, mod):
         data_shape = get_const_tuple(in_type.checked_type.shape)
         data_dim = len(data_shape)
         stride_dim = len(stride)
+        if data_dim == 0 and isinstance(inputs[0], _expr.Constant):
```

Comment:
I suspect this is `data_dim==1`, right? Otherwise, the data would just be empty. Also, could you add a test case  to check this situation? In general, every new feature should come with an appropriate test (see the guidelines : https://tvm.apache.org/docs/contribute/pull_request.html)

Reply:
Maybe I didn't describe it clearly, the input here is a single number. So `data_dim` here is 0. This situation is the same as `np.array(1)`. `len(np.array(1).shape) == 0`

Reply:
Cool, I didn't know that! I would still add a test to show when this situation arises (just to make sure that branch is covered). What do you think?

Reply:
👌add test for that.